src/proposals/20250220/SparkEthereum_20250220.t.sol

```diff
@@ -250,6 +253,12 @@ contract SparkEthereum_20250220Test is SparkTestBase {
     }
 
     function test_ETHEREUM_ARBITRUM_sparkLiquidityLayerE2E() public onChain(ChainIdUtils.Ethereum()) {
+        // Use mainnet timestamp to make PSM3 sUSDS conversion data realistic
+        uint256 mainnetTimestamp = block.timestamp;
+        chainSpellMetadata[ChainIdUtils.ArbitrumOne()].domain.selectFork();
+        vm.warp(mainnetTimestamp);
+        chainSpellMetadata[ChainIdUtils.Ethereum()].domain.selectFork();
```

Comment:
I'm pretty sure we can just revert to the old way of caching the mainnet timestamp from my PR and the convertToAssetValue will be exact again. I think when we switch back to arbitrums fork on like 274 it goes back to the non-mainnet timestamp. Should warp right after that fork to match mainnet timestamp.

Reply:
Good point. Fixed.